scripts/jobs/detached/00-v2-detached-jobs-deploy.ts

```diff
@@ -0,0 +1,81 @@
+import { ContractFactory } from 'ethers';
+import { run, ethers, network } from 'hardhat';
+import { e18, ZERO_ADDRESS } from '../../../utils/web3-utils';
+import * as contracts from '../../../utils/contracts';
+
+const { Confirm } = require('enquirer');
+const prompt = new Confirm('Do you wish to deploy v2 detached jobs contracts?');
+
+async function main() {
+  await run('compile');
+  await promptAndSubmit();
+}
+
+function promptAndSubmit(): Promise<void | Error> {
```

Comment:
We need to start implementing `hardhat-deploy` in this repo probably, also to avoid saving addresses manually in a json. Although OZ is coming up with a new library (can't pin point the name right now).